siteorigin-panels.php

```diff
@@ -56,6 +53,21 @@ function __construct() {
 		if( siteorigin_panels_setting( 'bundled-widgets' ) ) {
 			require_once plugin_dir_path( __FILE__ ) . 'widgets/widgets.php';
 		}
+
+		SiteOrigin_Panels_Widget_Shortcode::init();
```

Comment:
We definitely are. I often just make something a singleton out of habit, without actually thinking if it's singleton worthy or not. In a refactor, we should look at un-singling some singletons.